### DISCUSSION TITLE
Is there a reference currency?

### DISCUSSION THREAD

**Discussion by wsw70** [firefly-iii/firefly-iii#9156] [Firefly III] [answered]
I need to use two currencies in my budgeting (say EUR and GBP, EUR is my main one). Is there a way to reference the etotal worth in one currency?

Imagine I have 1000 EUR, and convert 500 EUR to 425 GBP. I will see a debit in my EUR account, and a credit in the GBP one. The accounts are disconnected, but the "total worth" has not changed. Despite this, in "My accounts" I will see that I went from 1000 EUR to 500 EUR.

I tried to locate a way to input a currency exchange rate to normalize everything to EUR, but found not find any.

What is the typical way in Firefly III to deal with multiple currencies and build an associated budget ?

**Comment by JC5** [comment, marked as answer]
Not yet, but this is something I am working on. No ETA I'm afraid.

**Reply by wsw70** [reply]
Thanks for the reply - this would be a great addition to help estimating a complete budget

**Reply by bandiba** [reply]
I very much support the idea! I actively use 3-4 different currencies, and the reports seem to show them separately, making them fragmented, hard to read. Converting to the reference currency would solve this.

**Reply by wsw70** [reply]
> I very much support the idea! I actively use 3-4 different currencies, and the reports seem to show them separately, making them fragmented, hard to read. Converting to the reference currency would solve this.

The main problem I have with split currencies is that I transfer, say, 100 EUR to CHF and should have no change in my "worth". But I see a decrease in EUR and an increase in CHF but since they are not linked the overall budget does not work.

**Reply by bandiba** [reply]
Right, that is slightly different problem from what I have, while introducing the reference currency for net worth would likely solve this one too.

**Reply by syonip** [reply]
Hi I'm also struggling with this.
Is it still impossible to show a unified picture in one currency?

**Reply by ProLucius** [reply]
> Not yet, but this is something I am working on. No ETA I'm afraid.

please tell me it's still in development, this would be a great feature and being able to set a fixed exchange rate would be awesome for things like stocks and gold and stuff

**Comment by JC5** [comment]
This has been implemented now, it's part of 6.2

https://docs.firefly-iii.org/explanation/financial-concepts/exchange-rates/

**Reply by ProLucius** [reply]
OMG THANK YOU!!!!!!!! 
Turns out i had to add a special docker variable in unraid for it to enable

**Reply by jacobburrell** [reply]
@JC5 can we enable a toggle in the GUI for this?

https://github.com/firefly-iii/firefly-iii/issues/11264
